connector_equipment/demo/backend_equipment_demo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
```suggestion
```
Not required in version 12...

Reply:
I don't have access to commit the suggestion...